app/models/searchgov_url.rb

```diff
@@ -13,7 +13,10 @@ class SearchgovUrl < ActiveRecord::Base
                                 application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
                               )
 
-  attr_accessible :last_crawl_status, :last_crawled_at, :url, :lastmod
+  attr_accessible :last_crawl_status,
+                  :last_crawled_at,
+                  :url, :lastmod,
```

Comment:
@peggles2, as long as you're breaking these into multiple lines, it would be good to separate `:url` and `:lastmod` into separate lines for readability.

Reply:
ack! missed that thanks.